supervisord.conf

```diff
@@ -69,7 +71,6 @@ command =
         cfg:default.server.root_url="%%(protocol)s://%%(domain)s:%%(http_port)s/graph"
 user = grafana
 directory = /usr/share/grafana
-environment = HOME=/usr/share/grafana
```

Comment:
This seemed to be redundant, since the same param is passed on [line L62](https://github.com/percona/pmm-server/pull/263/files#diff-500a0e8dcb2879d903640ba46031362cdc923d030e278cf2a681c24765cc87b0L62)